alignak/objects/macromodulation.py

```diff
@@ -64,25 +64,32 @@ class MacroModulation(Item):
 
     properties = Item.properties.copy()
     properties.update({
-        'macromodulation_name': StringProp(fill_brok=['full_status']),
-        'modulation_period': StringProp(brok_transformation=to_name_if_possible,
-                                        fill_brok=['full_status']),
+        'macromodulation_name':
+            StringProp(fill_brok=['full_status']),
+        'modulation_period':
+            StringProp(brok_transformation=to_name_if_possible, fill_brok=['full_status']),
     })
 
     running_properties = Item.running_properties.copy()
+    running_properties.update({
+        'customs':
+            StringProp(default={}, fill_brok=['full_status']),
```

Comment:
Yes, i copied/pasted ... from SchedulingItem or Contact that have exactly this awful code! I may give a try with a DictProp ...